analyzer/codechecker_analyzer/analyzers/clangsa/ctu_manager.py

```diff
@@ -140,14 +98,19 @@ def func_map_list_src_to_ast(func_src_list):
     return func_ast_list
 
 
-def map_functions(triple_arch, action, source, config, env,
-                  func_map_cmd, temp_fnmap_folder):
-    """ Generate function map file for the current source. """
-
+def map_functions_cmd(action, config, source, func_map_cmd):
+    """ Get command to create CTU index file. """
```

Comment:
I think using the extdef mapping naming here would be better to reflect what command is called. something like `get_extdef_mapping_cmd`